docs/01-programs/02-candy-machine/03-managing-candy-machines.md

```diff
@@ -1,12 +1,317 @@
+
+Note that once the first NFT has been minted, these attributes can no longer be updated.
+
+<Accordion>
+<AccordionItem title="JS SDK" open={true}>
+<div className="accordion-item-padding">
+
+Here’s an example of updating some of the shared NFT data on a Candy Machine. Any attributes that are not explicitly provided will stay the same.
+
+```tsx
+await metaplex.candyMachines().update({
+  candyMachine,
+  symbol: 'NEW',
+  sellerFeeBasisPoints: 100,
+  creators: [{ address: newCreator, share: 100 }],
+};
+```
+
+API References: [Operation](https://metaplex-foundation.github.io/js/classes/js.CandyMachineClient.html#update), [Input](https://metaplex-foundation.github.io/js/types/js.UpdateCandyMachineInput.html), [Output](https://metaplex-foundation.github.io/js/types/js.UpdateCandyMachineOutput.html), [Transaction Builder](https://metaplex-foundation.github.io/js/classes/js.CandyMachineBuildersClient.html#update).
+
+</div>
+</AccordionItem>
+</Accordion>
+
+## Update Collection
+
+Changing the Collection NFT associated with a Candy Machine is also possible. You’ll need to provide the address of the Collection NFT’s mint account as well as its update authority as a signer to approve this change.
+
+Note that, here also, once the first NFT has been minted, the collection cannot be changed.
+
+<Accordion>
+<AccordionItem title="JS SDK" open={true}>
+<div className="accordion-item-padding">
+
+To update the Collection NFT of a Candy Machine using the JS SDK you must provide a `collection` object containing the `address` of the Collection NFT’s mint account and the `updateAuthority` of the Collection NFT as a signer.
+
+```tsx
+await metaplex.candyMachines().update({
+  candyMachine,
+  collection: {
+    address: newCollection.address,
+    updateAuthority: newCollectionAuthority,
+  },
+};
+```
+
+API References: [Operation](https://metaplex-foundation.github.io/js/classes/js.CandyMachineClient.html#update), [Input](https://metaplex-foundation.github.io/js/types/js.UpdateCandyMachineInput.html), [Output](https://metaplex-foundation.github.io/js/types/js.UpdateCandyMachineOutput.html), [Transaction Builder](https://metaplex-foundation.github.io/js/classes/js.CandyMachineBuildersClient.html#update).
+
+</div>
+</AccordionItem>
+</Accordion>
+
+## Update Item Settings
+
+The item settings of a Candy Machine can also be updated but there are some limitations.
+
+- The **Items Available** attribute cannot be updated when using **Config Line Settings**.
+- The item settings cannot be updated such that we are swapping between **Config Line Settings** and **Hidden Settings**. However, if we’re not swapping the modes, the properties inside these settings can be updated.
+- Once the first NFT has been minted, these settings can no longer be updated.
```

Comment:
The `isSequential` can be updated as long as no NFT has been minted. After that, it is not possible.

Reply:
Isn't this covered by the sentence: "Once the first NFT has been minted, these settings can no longer be updated."?

Because I think none of these item settings can be updated once mint has started right?